lib/capybara/selenium/driver.rb

```diff
@@ -133,6 +133,10 @@ def evaluate_async_script(script, *args)
     unwrap_script_result(result)
   end
 
+  def send_keys(*args)
+    active_element.send_keys(*args)
```

Comment:
@twalpole there is [a test failure in CI](https://travis-ci.org/github/teamcapybara/capybara/jobs/743279918#L976-L985):

```
1) Capybara::Session with remote Chrome with selenium driver #send_keys defaults to sending keys to the document.activeElement

Failure/Error: raise Capybara::NotSupportedByDriverError, 'Capybara::Driver::Base#send_keys'

  Capybara::NotSupportedByDriverError:
    Capybara::Driver::Base#send_keys
  Shared Example Group: "Capybara::Session" called from ./spec/selenium_spec_chrome_remote.rb:73

# ./lib/capybara/driver/base.rb:63:in `send_keys'
# ./lib/capybara/session.rb:311:in `send_keys'
# ./lib/capybara/dsl.rb:58:in `block (2 levels) in <module:DSL>'
# ./spec/shared_selenium_session.rb:231:in `block (4 levels) in <top (required)>'
```

It seems like there are Selenium-enabled tests that _are not_ using the `Selenium::Driver`, and appear to be using `Driver::Base` or `RackTest::Driver`. Are the tests I've added missing a crucial setup step?

Reply:
@seanpdoyle You'll need to tag the test(s) as requiring send_keys  - like https://github.com/teamcapybara/capybara/blob/master/lib/capybara/spec/session/node_spec.rb#L1062 - although I am curious why the remote chrome driver version of the test would be failing.

Reply:
OH - I see the issue - your test isn't calling `send_keys` on the session - https://github.com/teamcapybara/capybara/pull/2422/files#diff-0794c7a2e4cba5e9448e4dfb1bd723358f79c2e7a1a3a0fc879c6a8e86e793c6R231

This is triggering the creation of a new default session which defaults to the RackTest driver

Reply:
🤦 , thank you!

With the way that the [DSL module delegates to `#page`](https://github.com/teamcapybara/capybara/blob/e8f7189730428ed9af04610ec4d1933424c6906c/lib/capybara/dsl.rb#L49-L62), will that style of invocation work in consumer applications' test suites (outside this test suite)?

Reply:
Yes -- it's just that in the tests the session is passed in rather than letting a default one be created since it makes it easier for testing against multiple drivers or with multiple sessions.  In most users case they're working with `Capybara.current_session` and wouldn't be passing the actual session around.